.github/workflows/release-please.yml

```diff
@@ -15,10 +15,4 @@ jobs:
         with:
           release-type: python
           token: ${{ secrets.GITHUB_TOKEN }}
-
-  publish-release:
```

Comment:
If this was intended to publish the tags related to a release, i.e. `vX`, `vX.Y` and `vX.Y.Z`, that would be a duplicate as it is handled directly in `container.yml` without needing another step to trigger it.
Another way would be to duplicate the bash script in `container.yml` in here but handling the `X`, `XY` and `XYZ` tags.